Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN apt-get update && apt-get install -y \
 ADD . /code/wasm-c-api
 WORKDIR /code/wasm-c-api
 RUN make v8-checkout && make -j v8
-RUN make C_COMP=gcc LD_FLAGS= C_FLAGS= 
+RUN make C_COMP=gcc LD_FLAGS= C_FLAGS="-fPIC"
```

Comment:
HEAD has modified this line, so you'll need to rebase this change.

Also, does it make sense to move this flag to the Makefile? Is there a reason not to always generate PIC?